pkg/query-service/README.md

```diff
@@ -28,5 +58,27 @@ go build -o build/query-service main.go
 ClickHouseUrl=tcp://localhost:9001 STORAGE=clickhouse build/query-service
 ```
 
+# Frontend Configuration for local query-service.
+
+- Set the following environment variables
+```console
+export FRONTEND_API_ENDPOINT=http://localhost:8080
+```
+
+- Update ${ENVIRONMENT.baseURL} from below line to localhost:9093
+https://github.com/SigNoz/signoz/blob/main/frontend/src/api/index.ts#L99
```

Comment:
please use from env here

Reply:
If you want me to add it here as follows
```
export const ENVIRONMENT = {
	baseURL:
		process?.env?.FRONTEND_API_ENDPOINT ||
		process?.env?.FRONTEND_ALERTMANAGER_ENDPOINT ||
                process?.env?.GITPOD_WORKSPACE_URL?.replace('://', '://8080-') ||
		'',
};

```
where FRONTEND_ALERTMANAGER_ENDPOINT=http://localhost:9093/
Then in that case it won't be picked up. Should I create another variable instead of used baseURL then?

Reply:
No another env is not feasible